src/subscript/runrms/runrms.py

```diff
@@ -81,6 +81,21 @@ def get_parser():
     # positional:
     prs.add_argument("project", type=str, nargs="?", help="RMS project name")
 
+    prs.add_argument(
+        "--project",
+        "-project",
+        dest="project_alt",
+        help="RMS project name (alternativ version)",
+    )
+
+    prs.add_argument(
+        "--list",
```

Comment:
`--listversions` instead?

Reply:
It is longer but probably better. Buty now add "-l" as a short version